api/src/opentrons/hardware_control/protocols/chassis_accessory_manager.py

```diff
@@ -60,3 +60,10 @@ def get_status_bar_state(self) -> StatusBarState:
 
         :returns: The current status bar state enumeration."""
         ...
+
+    def get_estop_state(self) -> EstopState:
+        """Get the current Estop state.
+        
+        If the Estop is not supported on this robot, this will always return Disengaged.
+
+        :returns: The current Estop state."""
```

Comment:
```suggestion
        :returns: The current Estop state."""
        ...
```
not strictly speaking necessary but we do it for the other methods

Reply:
d4f6c8017e0957d5b151bebc8e422f4f8014bdcb